src/locale/bootstrap-table-es-AR.js

```diff
@@ -20,6 +20,9 @@ $.fn.bootstrapTable.locales['es-AR'] = {
   formatDetailPagination (totalRows) {
     return `Showing ${totalRows} rows`
   },
+  formatClearSearch () {
+    return 'Clear Search'
```

Comment:
Let's use 'Limpiar búsqueda'

Reply:
Already commit by u (thanks!), so i resolve the conversations :)